docs/user/install.rst

```diff
@@ -133,6 +133,18 @@ For a more in-depth overview of available servers, see also:
 
     $ pip install [uvicorn|daphne|hypercorn]
 
+.. note::
+
+    By default, the ``uvicorn`` package comes only with a minimal set of
+    pure-Python dependencies.
+    For CPython-based production deployments, you can install Uvicorn along
+    with faster alternatives such as ``uvloop`` and ``httptools``::
+
+        $ pip install uvicorn[standard]
```

Comment:
This is an open question whether we should extend our test dependencies to include `[standard]`.
This seems to be a rather recent change, I swear `unicorn` used to pull in `uvloop` by default before.

Reply:
I think that a pipeline job may be useful, like the daphne and hypercorn ones.

Reply:
Hm, maybe we should add more environments in another _Action_ which would be run only upon merge to master (similar to _"tests emulated"_)? Like `uvicorn[standard]`, Windows+Cython, MacOS+Cython, Windows+Hypercorn etc. Fuller perf test suites could also be run in that fashion.

Reply:
Sure, that's an option, but it would duplicate the configuration somewhat since iirc you cannot share the yaml code.

Reply:
To that end, I'll make my upcoming ASGI E2E test suite use `uvicorn[standard]`.